vip-api/src/main/java/fr/insalyon/creatis/vip/api/security/oidc/OidcConfig.java

```diff
@@ -0,0 +1,86 @@
+package fr.insalyon.creatis.vip.api.security.oidc;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.core.io.Resource;
+import org.springframework.core.env.Environment;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.JsonNode;
+
+import java.util.ArrayList;
+import java.util.Collection;
+import java.io.FileInputStream;
+import java.net.URI;
+
+import fr.insalyon.creatis.vip.api.CarminProperties;
+
+@Configuration
```

Comment:
I think this could be a @Service (and it would be better)
@Configuration is used to declare other beans, which is not what we want.